vitrina/requests/views.py

```diff
@@ -44,3 +48,37 @@ def get_context_data(self, **kwargs):
         }
         context_data.update(extra_context_data)
         return context_data
+        
+        
+class RequestCreateView(LoginRequiredMixin, CreateView):
+    model = Request
+    form_class = RequestForm
+    template_name = 'base_form.html'
+
+    def form_valid(self, form):
+        self.object = form.save()
```

Comment:
```suggestion
        self.object = form.save(commit=False)
```

Since you are modifying `self.object` you need a `commit=False` and a `self.object.save()` call in the end.